src/interface/RHInterface.py

```diff
@@ -6,7 +6,7 @@
 import importlib
 import gevent # For threads and timing
 from gevent.lock import BoundedSemaphore # To limit i2c calls
-from monotonic import monotonic # to capture read timing
+from monotonic import monotonic # to capture read timing, pip install monotonic if there's an import error
```

Comment:
note: I had to install monotonic for whatever reason. Added this comment for anyone else even though it's not related to the ADS device.

Reply:
Probably best placed in the software setup docs.

Reply:
This is odd; monotonic is already in src/server/requirements.txt. No idea how it didn't get installed. I'll remove the comment. If anyone else has issues, it can be added to software setup docs.